recipes/cyipopt/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "cyipopt" %}
+{% set version = "0.1.7" %}
+{% set sha256 = "e62d4265743078c60f0af1426218310a9fbc1b6d0d0e60879e172d569dd3343d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/matthias-k/cyipopt/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # TODO: Once ipopt is available for Windows in conda forge, remove the
+  # following line.
+  skip: True  # [win]
+  # NOTE : Set CC to force gcc on OSX.
```

Comment:
IMO we should always use `gcc`, but some Mac people throw stones at me when I say that. You can do this but be aware that your package may be incompatible with the rest of the `conda-forge` stack that uses `clang`.